kubernetes/samples/scripts/common/jrf-domain-template.yaml

```diff
@@ -0,0 +1,124 @@
+# Copyright (c) 2017, 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+# This is an example of how to define a Domain resource.
+#
+apiVersion: "weblogic.oracle/v8"
+kind: Domain
+metadata:
+  name: %DOMAIN_UID%
+  namespace: %NAMESPACE%
+  labels:
+    weblogic.domainUID: %DOMAIN_UID%
+spec:
+  # The WebLogic Domain Home
+  domainHome: %DOMAIN_HOME%
+
+  # The domain home source type
+  # Set to PersistentVolume for domain-in-pv, Image for domain-in-image, or FromModel for model-in-image
+  domainHomeSourceType: %DOMAIN_HOME_SOURCE_TYPE%
+
+  # The WebLogic Server Docker image that the Operator uses to start the domain
+  image: "%WEBLOGIC_IMAGE%"
+
+  # imagePullPolicy defaults to "Always" if image version is :latest
+  imagePullPolicy: "%WEBLOGIC_IMAGE_PULL_POLICY%"
+
+  # Identify which Secret contains the credentials for pulling an image
+  %WEBLOGIC_IMAGE_PULL_SECRET_PREFIX%imagePullSecrets:
+  %WEBLOGIC_IMAGE_PULL_SECRET_PREFIX%- name: %WEBLOGIC_IMAGE_PULL_SECRET_NAME%
+
+  # Identify which Secret contains the WebLogic Admin credentials (note that there is an example of
+  # how to create that Secret at the end of this file)
+  webLogicCredentialsSecret: 
+    name: %WEBLOGIC_CREDENTIALS_SECRET_NAME%
+
+  # Whether to include the server out file into the pod's stdout, default is true
+  includeServerOutInPodLog: %INCLUDE_SERVER_OUT_IN_POD_LOG%
+
+  # Whether to enable log home
+  %LOG_HOME_ON_PV_PREFIX%logHomeEnabled: %LOG_HOME_ENABLED%
+
+  # Whether to write HTTP access log file to log home
+  %LOG_HOME_ON_PV_PREFIX%httpAccessLogInLogHome: %HTTP_ACCESS_LOG_IN_LOG_HOME%
+
+  # The in-pod location for domain log, server logs, server out, and Node Manager log files
+  %LOG_HOME_ON_PV_PREFIX%logHome: %LOG_HOME%
+  # An (optional) in-pod location for data storage of default and custom file stores.
+  # If not specified or the value is either not set or empty (e.g. dataHome: "") then the
+  # data storage directories are determined from the WebLogic domain home configuration.
+  dataHome: "%DATA_HOME%"
+
+
```

Comment:
Remove extra space

Reply:
Removed extra space in commit 686e19e. Thanks